### DISCUSSION TITLE
Is it possible to use maui as a sdk library in an ios/android app

### DISCUSSION THREAD

**Discussion by wnbotoo** [dotnet/maui#14223] [General]
Can I use maui as a aot library, no UI, just some common function such as network、database... which it can share with multiple platform? Just like [KMM](https://kotlinlang.org/docs/multiplatform-mobile-getting-started.html) (Kotlin Multiplatform Mobile).

C/CPP and Rust can do it, but I prefer C# because C# is a good language, I like it; .Net native AOT is release;

**Comment by xtuzy** [comment]
maui is ui framework, you can ask runtime team, see https://github.com/dotnet/runtime/issues/68775

**Comment by AndreasReitberger** [comment]
Yes, I also managed all my shared code in a maui Library.

https://github.com/AndreasReitberger/SharedMauiCoreLibrary
